regparser/web/jobs/models.py

```diff
@@ -25,3 +33,28 @@ class ParsingJob(models.Model):
         ("complete_with_errors", "complete_with_errors")
     ), default="received")
     url = models.URLField(blank=True, max_length=2000)
+
+    def save(self, *args, **kwargs):
+        super(ParsingJob, self).save(*args, **kwargs)
```

Comment:
Looks like that's a leftover vestige of some point where I had Redis-related code there; I'll remove that.